contracts/governance/gOHM.sol

```diff
@@ -124,13 +124,13 @@ contract gOHM is IgOHM {
      * @param _sOHM address
      */
     function migrate(address _staking, address _sOHM) external override onlyApproved {
-        require(_staking != approved);
+        require(_staking != approved, "Invalid argument");
 
-        require(_staking != address(0));
+        require(_staking != address(0), "Zero address found");
         approved = _staking;
 
-        require(address(sOHM) == address(0));
-        require(_sOHM != address(0));
+        require(address(sOHM) == address(0), "Cannot migrate twice");
+        require(_sOHM != address(0), "Zero address found");
```

Comment:
We should be explicit about where the zero-address was found.

Reply:
formatted as "Zero address: parameter"